main_test.go

```diff
@@ -88,7 +88,8 @@ var (
 // search for any request metrics between startTime and time.Now() for particular request methods
 // if testedmethods is empty, all metrics in timeframe are returned.
 func findMetricsInWindowForMethods(db database.PostgresClient, startTime time.Time, testedmethods []string) []database.ProxiedRequestMetric {
-	endTime := time.Now()
+	// add small buffer into future in case metrics are still being created
+	endTime := time.Now().Add(100 * time.Millisecond)
```

Comment:
could source from an environment variable and set to higher values in ci if tests are being flaky